apps/frontend/src/app/shared/components/cards/cards.component.html

```diff
@@ -1,34 +1,57 @@
 <div
-  class="group sm:w-[304px] w-[230px] h-[336px] bg-neutral mx-auto hover:shadow-lg flex justify-center relative rounded-lg"
+  class="sm:w-[304px] w-[230px] bg-neutral hover:shadow-lg mx-auto flex relative rounded-lg"
+  [ngClass]="[
+    details === 'more'
+      ? 'h-[280px] cursor-pointer'
+      : 'group  justify-center h-[336px]'
+  ]"
 >
-  <div class="font-mono sm:w-[304px] w-[230px]">
+  <div
+    class="font-mono"
+    [ngClass]="[details === 'more' ? 'w-[80px]' : 'sm:w-[304px] w-[230px]']"
+  >
     <img
       *ngIf="img"
       [src]="img"
-      class="h-40 w-full object-cover rounded-t-lg"
+      class="object-cover"
+      [ngClass]="[
+        details === 'more'
+          ? ' h-[80px] w-[80px] pl-4 pt-4 rounded'
+          : 'h-40 w-full rounded-t-lg'
+      ]"
       alt="image"
     />
     <img
       *ngIf="!img"
       [src]="'./assets/img/images/img-card-fallback.webp'"
-      class="h-40 w-full object-cover rounded-t-lg"
+      class="object-cover"
+      [ngClass]="[
+        details === 'more'
+          ? ' h-[80px] w-[80px] pl-4 pt-4 rounded'
+          : 'h-40 w-full rounded-t-lg'
+      ]"
       alt="fallback image"
     />
     <div
       class="px-4 absolute bottom-4 bg-neutral group-hover:-translate-y-8 duration-300 w-full"
     >
       <h3
-        class="text-base-100 font-sans font-medium text-2xl pb-1 pt-4 text-ellipsis overflow-hidden whitespace-nowrap"
+        class="text-base-100 pb-1 pt-4 text-ellipsis overflow-hidden whitespace-nowrap"
+        [ngClass]="[
+          details === 'more'
+            ? ' text-base font-mono font-bold'
+            : 'font-sans font-medium text-2xl'
+        ]"
       >
         {{ name }}
       </h3>
       <div class="pb-6">
-        <p class="text-base-200 font-mono h-[108px] line-clamp-2">
+        <p class="text-base-200 font-mono h-24 overflow-hidden line-clamp-3">
```

Comment:
Remember to apply these changes in https://github.com/Venture-Miner/wagademy/pull/63 too